### PR TITLE
refactor(frontend): replace V-Select with menu select from library

### PR DESCRIPTION
**Pull request by tewshi**
- replace vselect in csv import form with menu select
- replace vselect in managed assets form with menu select
- replace vselect in addressbook form with menu select
- fix space from comment here https://github.com/rotki/rotki/pull/7667#issuecomment-2020675671

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/components/asset-manager/managed/ManagedAssetForm.vue [hunk 1]
```diff
@@ -63,6 +65,21 @@ const isEvmToken = computed<boolean>(() => get(assetType) === EVM_TOKEN);
 
 const { allEvmChains } = useSupportedChains();
 
+const selectedAsetType = computed<SelectOption | undefined>({
+  get: () => !get(assetType) ? undefined : ({ key: get(assetType), label: toSentenceCase(get(assetType)) }),
+  set: value => set(assetType, value?.key),
+});
```
Comment (kelsos) [inline frontend/app/src/components/asset-manager/managed/ManagedAssetForm.vue:71]: Like I mentioned on discord I don't think this is a good approach, this wrapping seems really unnecessary. Can't we change the component to pass the `key-attr` value to the v-model directly?

**Review by kelsos — COMMENTED** [review]
So some initial comments

File: frontend/app/src/components/asset-manager/managed/ManagedAssetForm.vue (hunk 1 above)
Comment (tewshi) [reply frontend/app/src/components/asset-manager/managed/ManagedAssetForm.vue:71]: applying the change on the lib.

File: frontend/app/tests/e2e/pages/assets-manager-page.ts [hunk 2]
```diff
@@ -81,7 +81,7 @@ export class AssetsManagerPage {
     cy.get('[data-cy=asset-filter-menu]').should('exist');
     cy.get('[data-cy=asset-filter-none]').scrollIntoView();
     cy.get('[data-cy=asset-filter-none]').click();
-    cy.get('[data-cy=status-filter]').click();
+    cy.get('[data-cy=status-filter]').click({ force: true });
```
[the remark this answers is not in the document]
Comment (tewshi) [reply frontend/app/tests/e2e/pages/assets-manager-page.ts:84]: sometimes some overlay items cover other items on the ui, for instance a dropdown menu can cover an input field below it, cypress will not be able to focus on the input if this happens, the button click it supposed to toggle the menu, but in some test run, it doesn't close the menu (only in the test environment), the `force` option was a suggestion from the cypress.

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**